index.js

```diff
@@ -23,7 +23,7 @@ Nanostate.prototype.constructor = Nanostate
 
 Nanostate.prototype.emit = function (eventName) {
   var nextState = this._next(eventName)
-  assert.ok(nextState, `nanostate.emit: invalid transition ${this.state} -> ${eventName}`)
+  assert.ok(nextState, 'nanostate.emit: invalid transition', this.state, '->', eventName)
```

Comment:
`assert.ok` accepts a single string:
```suggestion
  assert.ok(nextState, 'nanostate.emit: invalid transition' + this.state + '->' + eventName)
```

Reply:
My bad, I'll update.